shrinkpdf.sh

```diff
@@ -64,16 +64,16 @@ shrink ()
 get_pdf_version() {
   # $1 is the input file. The PDF version is contained in the
```

Comment:
This PR still does not follow the style conventions of this project: tabs for indent, opening function parenthesis on a new line, etc. I know that style is very subjective and I don't personally have a problem with the common two-space indenting style, but the important thing is *consistency*. A project must use the same style throughout.

Reply:
Ah, sorry. I read over that. Will change it accordingly :+1: